app/uk/gov/hmrc/individualsdetailsapi/domains/integrationframework/IFAddress.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.individualsdetailsapi.domains.integrationframework
+
+import play.api.libs.functional.syntax._
+import play.api.libs.json.{Format, JsPath}
+import play.api.libs.json.Reads.{maxLength, minLength}
+
+case class IFAddress(
```

Comment:
I have realised I got the casing incorrect in my PR for domain models. 

They should all be like `IfAddress`

Maybe update them all?